compiler/AST/WhileStmt.cpp

```diff
@@ -206,9 +195,8 @@ void WhileStmt::checkConstLoops()
 
               else
               {
-                INT_FATAL(moveSrc,
-                          "Expected const loop condition variable to be "
-                          "true or false.");
+                // ... or without param folding
```

Comment:
I'm having a hard time understanding what this comment means based on the previous comment in the function.  It might be good to include the original start of the sentence, sorry :(